banzai/stages.py

```diff
@@ -18,7 +18,8 @@ def stage_name(self):
         return '.'.join([__name__, self.__class__.__name__])
 
     def run(self, images):
-        logger.info('Running {0}'.format(self.stage_name), image=images[0])
+        if len(images):
```

Comment:
I know it is implied but I like len(images)  > 0

Reply:
Fixed